lib/components/map/default-map.tsx

```diff
@@ -486,6 +503,7 @@ const mapStateToProps = (state) => {
     bikeRentalStations: state.otp.overlay.bikeRental.stations,
     carRentalStations: state.otp.overlay.carRental.stations,
     config: state.otp.config,
+    feeds: state.otp.transitIndex.feeds,
```

Comment:
What if transit index is undefined

Reply:
it won't be, it's defined in the initial state in the reducer.